cfpb/cfgov-refresh
# Ask disclaimer and date

Adds disclaimer and date to Ask answer pages

## Additions
- Last edited date on answer page
- Reusable text snippet `Legal disclaimer for educational materials` on Ask pages

## Testing

- Check out branch
- Set up Ask if necessary:
  ```
  export DEPLOY_ENVIRONMENT=build
  ./drop-db.sh
  ./create-mysql-db.sh
  mysql -uroot v1 < ~/Downloads/production_django.sql
  python cfgov/manage.py migrate
  python cfgov/manage.py runscript migrate_knowledgebase
  ```
- Navigate to [landing page](), [category page](), and [audience page]() and verify that legal disclaimer appears on all pages
- Navigate to [answer page](http://localhost:8000/ask-cfpb/how-quickly-can-i-get-money-after-i-deposit-a-check-into-my-checking-account-what-is-a-deposit-hold-en-1023/) and verify that last updated date and legal disclaimer appear on page

## Screenshots
### Answer page:
![new-ask-disclaimer](https://cloud.githubusercontent.com/assets/778171/26227139/570fa5b2-3be5-11e7-93cf-618f02c7ade0.png)

### Landing page:
![landing](https://cloud.githubusercontent.com/assets/778171/26517892/9d9b8500-4257-11e7-9d2e-15fb2b1db58d.png)

### Audience/Category pages:
![audience](https://cloud.githubusercontent.com/assets/778171/26517890/99ee2160-4257-11e7-8c31-f83aec6e5cab.png)



## Notes

- The disclaimer ReusableText snippet could be added to each Answer page individually along with the Feedback block in `get_or_create_page`, but I'm inserting it directly in the template so that if something other than its text needs to be changed, it can all be done in one place.

## Checklist
